### PR TITLE
Add getAccessTokenExpirationTime and bump sdkVersion

### PR DESCRIPTION
**Pull request by jorunfa**

### CODE REVIEW

File: connect/src/com/telenor/connect/id/ConnectIdService.java [hunk 1]
```diff
@@ -58,6 +58,14 @@ public String getAccessToken() {
         return retrieveTokens().getAccessToken();
     }
 
+    public Date getAccessTokenExpirationTime() {
+        if (retrieveTokens() == null) {
```
Comment (stephan-nordnes-eriksen) [inline connect/src/com/telenor/connect/id/ConnectIdService.java]: Maybe cache result to variable so you don't call retrieveTokens twice?

Comment (jorunfa) [inline connect/src/com/telenor/connect/id/ConnectIdService.java]: I see what you mean. `retrieveTokens()` uses a cache and only actually fetches the tokens from the store if they are missing. It's a private method.

File: connect/src/com/telenor/connect/id/ConnectIdService.java [hunk 2]
```diff
@@ -58,6 +58,14 @@ public String getAccessToken() {
...
+            return null;
+        }
+
+        return retrieveTokens().getExpirationDate();
```
Comment (jorunfa) [inline connect/src/com/telenor/connect/id/ConnectIdService.java]: See above.

Comment (jorunfa) [inline connect/src/com/telenor/connect/id/ConnectIdService.java]: If I'm not misunderstanding something the `getString(…)` from `getSharedPreferences` also uses a cache internally, and blocks (is synchronous), so there shouldn't be race conditions here, I think?

File: connect/src/com/telenor/connect/id/ConnectIdService.java (hunk 1 above)
Comment (stephan-nordnes-eriksen) [inline connect/src/com/telenor/connect/id/ConnectIdService.java]: Yes, but you should probably do something like
```
DataType datatype = retrieveTokens();
if(datatype == null){
//...

return datatype.getExpirationDate();
```

ps. I don't know what datatype should be used.

**Pull request merged**